frontend/src/components/RoundResultScreen.js

```diff
@@ -1,6 +1,43 @@
-const RoundResultsScreen = ({ players, roundResult }) => {
-  // TODO display current leader board and also indicate if the current user got the answer correct
-  return <div>Round Results Screen</div>;
+import React, { useState, useEffect } from "react";
+import "./RoundResultScreen.css";
+
+const RoundResultsScreen = ({ players, roundResult, currentUserPlayerId, timer }) => {
+    const [timeLeft, setTimeLeft] = useState(timer);
+    useEffect(() => {
+        setTimeLeft(timer);
+
+        const interval = setInterval(() => {
+            setTimeLeft(prevTimeLeft => Math.max(prevTimeLeft - 1, 0));
+        }, 1000);
+
+        return () => {
+            clearInterval(interval);
+        };
+    }, [timer]);
+
+    const sortedPlayers = players?.slice().sort((a, b) => b.points - a.points);
```

Comment:
Optional but I'm thinking we sort the players on the backend before sending the list of players to the frontend. This way,  we only have to sort once vs every client sorting. See my other thread about sorting.

Reply:
I saw you sorted in the backend, so we don't need to sort this here anymore

Reply:
yes you are right, forgot to delete. Deleted now